packages/mobile/package.json

```diff
@@ -117,11 +117,9 @@
     "react-native-screens": "^2.7.0",
     "react-native-secure-randombytes": "^3.0.2",
     "react-native-send-intent": "git+https://github.com/celo-org/react-native-send-intent#a0f4b00",
-    "react-native-shadow": "^1.2.2",
```

Comment:
We used to use this to get more consistent shadow behavior on iOS and android. Can you confirm it's no longer needed? 
Always glad to see a dep removed, just want to confirm.

Reply:
Yes in the few places where we use a shadow, using `elevation` was good enough on Android to mimic the more controllable shadow supported on iOS, though indeed they don't look exactly the same.